content/en/research/_index.md

```diff
@@ -0,0 +1,31 @@
+{
+  "title": "Research documentation",
+  "weight": 0,
+  "bookCollapseSection": true,
+  "bookFlatSection": true,
+  "cascade": [
+        {
+            "type": "research",
+            "_target": {
+                "path": "/research/**"
+            }
+        }
+    ]
+}
+
+# Research documentation
+
+In the documentation for the tracker adapters, we attribute types of data to paths in the decoded requests. These attributions are the result of our research and we provide the reasons in the tables in the tracker documentation and in the adapter objects in TrackHAR (in the `reasoning` field), so that others don’t have to trust our results blindly. We decided on a finite set of justification that we will give listed here in the order of importance that we will go through when choosing a reason:
+
+1. **Public documentation**  
+  Sometimes, we might find documentation by the tracking company itself on which data is being transmitted. If this is the case, we will link to the relevant sections of this documentation instead of giving a simple reason for why we classified a particular path.
+2. **Public tracker source code**  
+  If the tracking software is open source, we will analyze the source code and try to find the lines showing what data is transmitted in the end. We will explain that in a short write-up of our investigation, contained in this section and linked to from the adapter.
```

Comment:
I fear we'll also have to resort to decompilation sooner rather than later.

Reply:
do you want me to also mention that?

Reply:
Just be less specific, we'll analyze the SDK, OS or not.